pkg/cloudclient/add_aws.go

```diff
@@ -0,0 +1,64 @@
+package cloudclient
+
+import (
+	"context"
+	"github.com/openshift/osd-network-verifier/pkg/cloudclient/aws"
+	"os"
+)
+
+func init() {
+	Register(
+		aws.ClientIdentifier,
+		produceAWS,
+	)
+}
+
+var (
+	DefaultTags     = map[string]string{"osd-network-verifier": "owned", "red-hat-managed": "true", "Name": "osd-network-verifier"}
+	RegionEnvVarStr = "AWS_REGION"
+	RegionDefault   = "us-east-1"
+)
+
+// Precedence: cli > env var > default
+func getRegion(options CmdOptions) string {
+	if options.Region != "" {
+		return options.Region
+	}
+	val, present := os.LookupEnv(RegionEnvVarStr)
+	if present {
+		return val
+	} else {
+		return RegionDefault
+	}
+}
+
+func produceAWS(options *CmdOptions) (CloudClient, error) {
```

Comment:
I'm not sure what this function does just by reading the name here. Maybe add a comment explaining what the goal is for this function?